docs/admin/logical-replication.rst

```diff
@@ -48,26 +48,28 @@ A publication is the upstream side of logical replication and it's created on
 the cluster which acts as a data source.
 
 Each table can be added to multiple publications if needed. Publications can
-only contain tables. All operation types (INSERT, UPDATE, DELETE and schema
-changes) are replicated.
+only contain tables. All operation types (``INSERT``, ``UPDATE``, ``DELETE`` and
+schema changes) are replicated.
 
 Every publication can have multiple subscribers.
 
 A publication is created using the
 :ref:`CREATE PUBLICATION <sql-create-publication>` command and may later be
 altered or dropped using corresponding commands. The individual tables can be
 added and removed dynamically using
-:ref:`ALTER PUBLICATION <sql-alter-publication>`
+:ref:`ALTER PUBLICATION <sql-alter-publication>`.
 
 .. CAUTION::
 
-    Publishing cluster must have
-    :ref:`sql-create-table-soft-deletes-enabled` setting
-    set to ``true`` so that subscribing cluster can catch up all changes made
-    during replication pause caused by network issues or explicitly done by a user.
+    The publishing cluster must have
+    :ref:`sql-create-table-soft-deletes-enabled`
+    set to ``true`` so that a subscribing cluster can catch up with all changes
+    made during replication pauses caused by network issues or explicitly done by
+    a user.
```

Comment:
Is it `true` by default?

Reply:
Yes, it's ``true`` by default. At least from what the `soft_deletes.enabled` documentation says, as well as what `SHOW CREATE TABLE` returns.